### DISCUSSION TITLE
ISO 27002:2022 knowledge base

### DISCUSSION THREAD

**Discussion by dbarzin** [monarc-project/MonarcAppFO#499] [General]
With Monarc 2.12.6, after importing the referential ISO 27002:2022, the controls are not present in the information risks knowledge base  :

![image](https://github.com/monarc-project/MonarcAppFO/assets/43208232/93e0a06e-08cc-42bc-8794-abf999cb091a)

as describe in the documentation : https://www.monarc.lu/documentation/user-guide/#risks

How can I get the controls for ISO 27002:2022 in the information risks knowledge base ?

**Comment by ruslanbaidan** [comment]
Please follow the referenced video guide:
https://github.com/monarc-project/MonarcAppFO/discussions/361#discussioncomment-2233929

**Comment by dbarzin** [comment]
I have followed the video guide but I still do not have the controls. Even if I start a new risk analysis with ISO 27001:2013

![image](https://github.com/monarc-project/MonarcAppFO/assets/43208232/da986335-0ce8-426e-bf9c-414a9c498415)
